WeakAuras/RegionTypes/DynamicGroup.lua

```diff
@@ -126,6 +134,55 @@ local function create(parent)
   return region
 end
 
+local function noop() end
+
+local anchorers = {
+  self = function(self, anchorLocation) return self end,
+  nameplate = function(self, anchorLocation) return WeakAuras.GetUnitNameplate(anchorLocation) end,
+  unit = function(self, anchorLocation) return WeakAuras.GetUnitFrame(anchorLocation) end,
```

Comment:
This is unfourtunately a bit more involved than this.

Since some unit frames addons (Vuhdo!) have a delay until they create a frame matching a unit, the LibGetFrame library has a callback interface where we are informed if a unit -> frame mapping changed.

That's the code in the old dynamic group involving dyngroup_unitframe_monitor, which does need some massaging to make work. The existing code probably needs some changes to support you too. @mrbuds wrote that part of the code (and LibGetFrame).

Reply:
LibGetFrame fire 3 callbacks https://github.com/mrbuds/LibGetFrame/blob/master/LibGetFrame-1.0.lua#L134
`GETFRAME_REFRESH` after each scan of all children of UIParent
`FRAME_UNIT_UPDATE => frame, unit` when a `frame` is associated with a new `unit`
`FRAME_UNIT_REMOVED => frame, unit` when `unit` is no longer associated with `frame`